folksonomy/api.py

```diff
@@ -81,11 +81,7 @@ async def app_lifespan(app: FastAPI):
     # So, for everything to work correctly, it's better to specify explicitly the allowed origins."
     # => Workarround: use allow_origin_regex
     # Source: https://github.com/tiangolo/fastapi/issues/133#issuecomment-646985050
-    allow_origins=[
-        "http://localhost:8000",
-        "http://world.openfoodfacts.localhost",
-    ],
-    allow_origin_regex="https?://.*",
+    allow_origin_regex=r"^https://.*|^http://(localhost|.*\.localhost)(:\d+)?$",
```

Comment:
It's absolutely not what we want @kirtanchandak ! Here you are limiting to localhost based instances… disabling production instances !

Reply:
Or are they http:// internally?